src/ui/Views/Course/Variants.cshtml

```diff
@@ -1,94 +1,174 @@
 ﻿@model FromUcasViewModel
 
 @{
-    Layout = "_Layout";
-    ViewBag.Title = Model.Course.Name;
-    var link = "mailto:becomingateacher@digital.education.gov.uk?subject=Problem%20with%20course%20" + Model.Course.ProgrammeCode;
+   Layout = "_Layout";
+   ViewBag.Title = Model.Course.Name;
 }
 
-<a href="@Url.Action("Courses", "Organisation", new {ucasCode = Model.Course.UcasCode})" class="govuk-back-link">Back</a>  
+<a href="@Url.Action("Courses", "Organisation", new {ucasCode = Model.Course.UcasCode})" class="govuk-back-link">Back</a>
 
 <main role="main" class="govuk-main-wrapper" id="main-content">
-    <h1 class="govuk-heading-xl">
-        <span class="govuk-caption-xl">@Model.Course.Name</span>
-        @Model.Course.Type (@Model.Course.ProgrammeCode)
-    </h1>
+  <h1 class="govuk-heading-xl">
+    <span class="govuk-caption-xl">@Model.Course.Name</span>
+    @Model.Course.Type (@Model.Course.ProgrammeCode)
```

Comment:
Course type and name should be the other way around now:
![screen shot 2018-08-03 at 13 36 06](https://user-images.githubusercontent.com/319055/43643157-36b16de8-9722-11e8-942f-27cd244119b4.png)